### PR TITLE
Upgrade dependencies.

### PR DESCRIPTION
**Pull request by notlee**
This commit updates colour usecases as part of the o-colors upgrade.
It also consolidates duplicated demo templates.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "o-share",
   "dependencies": {
-    "ftdomdelegate": "^3.1.0",
-    "o-colors": "^4.0.2",
-    "o-normalise": "^1.2.1",
+    "ftdomdelegate": "#breaking",
```
Comment (JakeChampion) [inline bower.json]: Origami needs to have versions which work in bower and npm. `#breaking` does not work in npm, npm requires one of these two formats:
- `git://github.com/<user>/<project>.git#<branch>`
- `<user>/<project>#<branch>` (If using github)

Comment (notlee) [reply bower.json]: This won't be published until these are updated. My intention is to ignore the occ error until the low level components are stable -- then I'll release betas again.

Comment (notlee) [reply bower.json]: Okay I did end up going back to betas. Seeing all the failing builds was too much (and hard to tell if it was just occ or an actual issue). Hopefully I won't need to release another low level beta.

**Comment by origamiserviceuser** [comment]
o-share bundle size difference from 6.5.0 to 21818667f2209a49317e164cf11fe61774e56c2a
js: 0.75kb increase (0.23kb/gzip)
css: 4.05kb decrease (-0.11kb/gzip)

**Pull request merged**